src/converters/convert.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Given an input is a converter, it will return a converted result.
+ *
+ * @param {object} input - The input to be converted.
+ * @param {boolean} defaultValue - Default value to use if conversion fails.
+ * @param {string} name - The name of of what is converted.
+ *
+ * @returns {object} - The converted result.
+ */
```

Comment:
The most correct approach would be @returns {string|object[]|bool|number|RegExp} or undefined. Not sure if it should be a more generic return, or just write everything.